devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -443,6 +448,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         return false;
     }
 
+    if (!(config.check("jointList") && config.find("jointList").isList())) {
+        yInfo() << LogPrefix << "jointList option not found or not valid, all the model joints are selected.";
+        pImpl->jointList.clear();
```

Comment:
Do you think in this case where the joints were not specified, it would be helpful to return the joints list used by the loaded model back to the user? This can help reduce uncertainties for logging and testing with some data.

Reply:
I think some information are given below when the model is loaded. This device is already quite verbose, so I would avoid adding further messages (I would rather try to remove some of them in the future).